src/openrct2/world/park.c

```diff
@@ -70,10 +70,7 @@ sint32 _suggestedGuestMaximum;
  */
 sint32 _guestGenerationProbability;
 
-sint16 gParkEntranceX[MAX_PARK_ENTRANCES];
-sint16 gParkEntranceY[MAX_PARK_ENTRANCES];
-sint16 gParkEntranceZ[MAX_PARK_ENTRANCES];
-uint8 gParkEntranceDirection[MAX_PARK_ENTRANCES];
+rct_xyzd16 gParkEntrance[MAX_PARK_ENTRANCES];
```

Comment:
Almost all static arrays in the code base use the plural form, `gParkEntrances` would fit better.